.prettierrc

```
@@ -0,0 +1 @@
+{}
```

I'd suggest renaming to `prettier.config.mjs` for some Intellisense and adding [@prettier/plugin-oxc](https://github.com/prettier/prettier/tree/main/packages/plugin-oxc) for mad speed, which was [announced](https://prettier.io/blog/2025/06/23/3.6.0#change-17472) alongside Prettier 3.6.

After running:

```sh
npm add -D prettier@latest @prettier/plugin-oxc
```

```suggestion
/**
 * @satisfies {import("prettier").Config}
 * @see https://prettier.io/docs/en/configuration.html
 */
const config = {
  plugins: ["@prettier/plugin-oxc"],
};

export default config;
```